test_util/test_installer_ccm.py

```diff
@@ -324,8 +324,6 @@ def establish_host_connectivity():
             vpc.delete()
     else:
         log.info("Test failed! VPC will remain for debugging 1 hour from instantiation")
-    if options.ci_flags:
```

Comment:
@mellenburg The suppressing the test return code is absolutely necessary for us to be able to mute / unmute test. Assumes you have "smart" CI which can interpret the results of the test report (Which is what TeamCity does). If people aren't in CI, they get the return code.

The build is setup to fail on non-zero error code _or_ non-passing test results. If there are non-passing test results, and that also leads to a non-zero return, then you can't mute tests in CI. See the PR which added this for more explanation.